### PR TITLE
Machine Learning Integrated

### PR DESCRIPTION
**Pull request by edwinPato**
Machine Learning integrated with end Point. End Point POST can upload a .zip file and unzip the images. Also, the end point returns the necesary data for our UI.

### CODE REVIEW

File: CONVERTER_SERVICE/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (jchoque87) [inline CONVERTER_SERVICE/.gitignore:1]: you should add a break line in this file

File: MLendPoint/src/controllers/CocoSSD.js [hunk 2]
```diff
@@ -80,8 +80,13 @@ class CocoSSD extends require("./MachineLearing") {
...
         let arrParse = new Array();
         arr.forEach((element) => {
-            let arrAux = { className: element.class, probability: element.score };
-            arrParse.push(arrAux);
+            //let arrAux = { "className": element.class, "probability": element.score };
+            const obj = {
+                className: element.class,
+                probability: element.score,
+            };
+            const auxi = JSON.stringify(obj);
```
Comment (jchoque87) [inline MLendPoint/src/controllers/CocoSSD.js:88]: You are use JSON.parse in the route. It's necessary use JSON.stringify in this line. I think is double job

File: MLendPoint/src/middleware/unzip.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+const AdmZip = require("adm-zip");
+const path = require("path");
+
+class UnZip {
+    constructor() { }
+
+    extractZip(zipPath, outPath) {
```
Comment (jchoque87) [inline MLendPoint/src/middleware/unzip.js:7]: This function has keywords how to let, const and var. This last is not recommended for new uses

File: MLendPoint/src/middleware/secondToString.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
+
+class SecondsToString{
+    
+    constructor() {}
+
+    parse(seconds) {
```
Comment (jchoque87) [inline MLendPoint/src/middleware/secondToString.js:6]: The keyword var is not recommended for ES6

**Review by jchoque87 — COMMENTED** [review]
You should use only let or const for the code in ES6

**Review by Milena-Rodriguez-Montecinos — APPROVED** [review]

**Pull request merged**